build/includes/sdk.mk

```diff
@@ -213,3 +213,17 @@ sdk-shell-csharp:
 sdk-publish-csharp: RELEASE_VERSION ?= $(base_version)
 sdk-publish-csharp:
 	$(MAKE) run-sdk-command-csharp COMMAND=publish VERSION=$(RELEASE_VERSION) DOCKER_RUN_ARGS="$(DOCKER_RUN_ARGS) -it"
+
+# difference in sdks before and after gen-all-sdk-grpc target
+test-gen-all-sdk-grpc:
+	mkdir -p includes/tmp
+	cp -r ../sdks includes/tmp
```

Comment:
It's also the `test/sdk` directory.

Reply:
I have included the 'test/sdk' directory to compare the differences.